apps/ewallet_db/lib/ewallet_db/role.ex

```diff
@@ -25,15 +26,26 @@ defmodule EWalletDB.Role do
 
   defp changeset(%Role{} = key, attrs) do
     key
-    |> cast(attrs, [:name, :display_name])
-    |> validate_required(:name)
+    |> cast(attrs, [:priority, :name, :display_name])
+    |> validate_required([:name, :priority])
```

Comment:
Curious question: `:priority` can also be a negative integer right?

Reply:
Yup!